.github/workflows/checks.yaml

```diff
@@ -41,11 +41,7 @@ jobs:
         shell: "sh"
         run: "./lint.sh"
   check_features:
-    runs-on: "ubuntu-latest"
-    steps:
-      - uses: "actions/checkout@v3"
-      - uses: "nolus-protocol/rust-ci-actions/rust_cache@main"
-      - uses: "nolus-protocol/rust-ci-actions/check_features@main"
```

Comment:
don't you have to cleanup 'nolus-protocol/rust-ci-actions/check_features' then?

Reply:
It should be updated, yes, but not removed as it's used within `check_features.yaml`, which is in turn called by `checks.yaml` /this file/.